src/main/java/io/bf2/kafka/common/Config.java

```diff
@@ -173,4 +178,19 @@ public class Config {
             .define(TIMEOUT_SECONDS, ConfigDef.Type.INT, DEFAULT_TIMEOUT_SECONDS,ConfigDef.Importance.MEDIUM, "Timeout duration for listing and describing topics")
             .define(SCHEDULE_INTERVAL_SECONDS, ConfigDef.Type.INT, DEFAULT_SCHEDULE_INTERVAL_SECONDS,ConfigDef.Importance.MEDIUM, "Schedule interval for scheduled counter");
     // ===== end of partition counter configs definition =====
+
+    public static final String DEFAULT_REPLICATION_FACTOR = KafkaConfig.DefaultReplicationFactorProp();
+    public static Optional<Short> brokerMinInsyncReplicas(Map<String, ?> configs) {
+        return getConfig(MIN_IN_SYNC_REPLICAS_CONFIG, configs)
+                .map(v -> Short.valueOf(v.toString()));
+    }
+
+    public static Optional<Short> brokerDefaultReplicationFactor(Map<String, ?> configs) {
+        return getConfig(DEFAULT_REPLICATION_FACTOR, configs)
+                .map(v -> Short.valueOf(v.toString()));
+    }
+
+    public static Optional<Object> getConfig(String name, Map<String, ?> configs){
+        return Optional.ofNullable(configs.get(name));
+    }
```

Comment:
Put these methods into common/Config because they are used in create topic policy and alter config policy, and they are config related methods.